docs/en/latest/FAQ.md

```diff
@@ -167,7 +168,7 @@ spec:
 
 </TabItem>
 
-<TabItem value="2.15">
+<TabItem value="3.1.0-debian">
```

Comment:
The configmap configuration here is incorrect. The 3.x.x version uses the 2.x.x version. I think you should install and verify it.